filebeat/docs/multiple-prospectors.asciidoc

```diff
@@ -18,8 +23,12 @@ filebeat:
     -
       paths:
         - "/var/log/apache2/*"
+      fields:
+        json: true
+      fields_under_root: true
 -------------------------------------------------------------------------------------
 
-The config file in the example starts two prospectors. The first prospector has two harvesters,
+The config file in the example starts two prospectors (each prospector begins with a `-`). The first prospector has two harvesters,
```

Comment:
not sure I like this phrase, as it pretty much sounds like magic:

> (each prospector begins with a `-`)

maybe we should stress the fact config files  are YAML and `prospectors:` requires a YAML array which normally is written by having every element beginning with `-`

@ruflin changed configs to have `input_type: log` right after the `-` (no empty new line). The docs should do the same (for some reason people like to remove the `-` itself)